src/compiler/comments.ts

```diff
@@ -127,15 +132,21 @@ namespace ts {
             function getTrailingComments(range: TextRange): CommentRange[];
             function getTrailingComments(range: TextRange, contextNode: Node, ignoreNodeCallback: (contextNode: Node) => boolean, getTextRangeCallback: (contextNode: Node) => TextRange): CommentRange[];
             function getTrailingComments(range: TextRange, contextNode?: Node, ignoreNodeCallback?: (contextNode: Node) => boolean, getTextRangeCallback?: (contextNode: Node) => TextRange) {
+                let ignored = false;
                 if (contextNode) {
                     if (ignoreNodeCallback(contextNode)) {
-                        return undefined;
+                        ignored = true;

```

Comment:
It does not. This is left over from gathering some individual performance metrics and just needs to be cleaned up.